src/Tizen.NUI/src/public/Utility/Capture.cs

```diff
@@ -65,32 +67,75 @@ internal Capture(IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override void Dispose(DisposeTypes type)
+        /// <since_tizen> 9 </since_tizen>
+        public Position Position { get; set; } = new Position(0, 0, 0);
+
+        /// <summary>
+        /// Size of capturing region
+        /// Size value for each direction is required larger than zero.
+        /// Default is window size.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Size Size { get; set; } = new Size(0, 0, 0);
+
+        /// <summary>
+        /// image file path to be saved as a file.
+        /// If path is empty string, the captured result is not be saved as a file.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string Path { get; set; } = null;
+
+        /// <summary>
+        /// background color of captured scene
+        /// Default color is Color.Transparent
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Color ClearColor { get; set; } = Color.Transparent;
+
+        /// <summary>
+        /// The value to control image quality for jpeg file format in the range [1, 100]
+        /// Default is 100.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public uint Quality
         {
-            if (disposed)
+            get
             {
-                return;
+                return quality;
             }
-
-            if (type == DisposeTypes.Explicit)
+            set
             {
-                //Called by User
-                //Release your own managed resources here.
-                //You should release all of your own disposable objects here.
+                quality = Math.Min(value, 100);
+                Interop.Capture.SetImageQuality(SwigCPtr, quality);
             }
-
-            base.Dispose(type);
         }
 
-        /// This will not be public opened.
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override void ReleaseSwigCPtr(HandleRef swigCPtr)
+        /// <summary>
+        /// Start capture and save the image as a file.
+        /// </summary>
+        /// <param name="source">source View or Layer to be used for capture.
+        /// <since_tizen> 9 </since_tizen>
+        public void Start(Container source)
         {
-            Interop.Capture.Delete(swigCPtr);
+            Size size = Size;
+            string path = Path;
+            if(size.Width == 0 || size.Height == 0 || size.Depth == 0)
+            {
+                size = NUIApplication.GetDefaultWindow().Size;
+            }
+            if (null == path)
+            {
+                path = "";
+            }
+            if (source is View || source is Layer)
+            {
+                Interop.Capture.Start4(SwigCPtr, source.SwigCPtr, Position.SwigCPtr, new Vector2(size.Width, size.Height).SwigCPtr, path, new Vector4(ClearColor.R, ClearColor.G, ClearColor.B, ClearColor.A).SwigCPtr);
+
```

Comment:
Please remove the build warnings.
-> Call System.IDisposable.Dispose on object created by 'new Vector2(size.Width, size.Height)' before all references to it are out of scope.

You can define Vector2 and Vector4 instances separately.

Reply:
Thank you x2